modules/core/04-channel/keeper/grpc_query_test.go

```diff
@@ -1219,6 +1229,45 @@ func (suite *KeeperTestSuite) TestQueryUnreceivedPackets() {
 			},
 			true,
 		},
+		{
+			"success basic unreceived packet commitments, ordered channel",
+			func() {
+				path := ibctesting.NewPath(suite.chainA, suite.chainB)
+				path.SetChannelOrdered()
+				suite.coordinator.Setup(path)
+
+				suite.chainA.App.GetIBCKeeper().ChannelKeeper.SetNextSequenceRecv(suite.chainA.GetContext(), path.EndpointA.ChannelConfig.PortID, path.EndpointA.ChannelID, 1)
+
+				expSeq = []uint64{1}
+				req = &types.QueryUnreceivedPacketsRequest{
+					PortId:                    path.EndpointA.ChannelConfig.PortID,
+					ChannelId:                 path.EndpointA.ChannelID,
+					PacketCommitmentSequences: []uint64{1},
+				}
+			},
+			true,
+		},
+		{
+			"success basic multiple unreceived packet commitments, ordered channel",
+			func() {
+				path := ibctesting.NewPath(suite.chainA, suite.chainB)
+				path.SetChannelOrdered()
+				suite.coordinator.Setup(path)
+
```

Comment:
```
path := ibctesting.NewPath(suite.chainA, suite.chainB)
				path.SetChannelOrdered()
				suite.coordinator.Setup(path)
```
maybe this can be set up in the base test function below, rather than replicated in each test?

Reply:
half the cases don't currently do this set up thankfully and half of the ones that do set up an unordered channel so I'd guess it would be more trouble than its worth if that makes sense?